src/main/kotlin/com/openlattice/mechanic/upgrades/AddSchemaToExternalTables.kt

```diff
@@ -99,18 +99,18 @@ class AddSchemaToExternalTables(
 
         val tableIds = externalTables.keys.toSet()
 
-        edms.deleteOrganizationExternalDatabaseColumnObjects(externalColumns
-                .values
-                .toList()
-                .groupBy { it.tableId }
-                .filter { tableIds.contains(it.key) }
-                .mapValues {
-                    it.value
-                            .filter { c -> orgIds.contains(c.organizationId) }
-                            .map { c -> c.id }
-                            .toSet()
-                }
-        )
+//        edms.deleteOrganizationExternalDatabaseColumnObjects(externalColumns
+//                .values
+//                .toList()
+//                .groupBy { it.tableId }
+//                .filter { tableIds.contains(it.key) }
+//                .mapValues {
+//                    it.value
+//                            .filter { c -> orgIds.contains(c.organizationId) }
+//                            .map { c -> c.id }
+//                            .toSet()
+//                }
+//        )
```

Comment:
doesn't really matter since this migration only needed to be run once and is basically forever retired now, but if we're leaving retired migrations around for the historical value, it seems strictly better to leave the code that was actually run intact.